src/modules/roc_core/target_posix/roc_core/time.cpp

```diff
@@ -49,6 +52,117 @@ void sleep_for_ms(uint64_t ms) {
         }
     }
 }
+#elif defined(__APPLE__) && defined(__MACH__)
```

Comment:
We need a separate target directory for macos-specific code, because it has nothing to do with posix. IIRC we already have target_darwin.